uiowa/uiowa
# Updated to latest version of acquia/blt

# How to test
- [Review code changes](https://github.com/uiowa/uiowa/pull/6144/files) - most of them are from PHPCS errors that popped up after switching to the new rules.
- Confirm build completes
- Run `ddev blt ds` to test a sync.
- Run any other blt commands you care to test out.
- You should still see twig debug info on your local sites.
